src/UCTSearch.h

```diff
@@ -73,12 +73,12 @@ class UCTSearch {
     static constexpr passflag_t NORESIGN = 1 << 1;
 
     /*
-        Maximum size of the tree in memory. Nodes are about
-        48 bytes, so limit to ~1.2G on 32-bits and about 5.5G
-        on 64-bits.
+        Maximum size of the tree in memory. Nodes are 56 bytes, edges are
```

Comment:
sizeof(UCTNode) == 64 for me. I count 60 bytes by adding up the variables so I believe there is either 4 bytes padding between m_has_edges and m_edges or there is spare at the end